SingularityService/src/main/java/com/hubspot/singularity/views/IndexView.java

```diff
@@ -239,6 +239,10 @@ public String getRedirectOnUnauthorizedUrl() {
     return redirectOnUnauthorizedUrl;
   }
 
+  public boolean isShortenSlaveUsageHostname() {
```

Comment:
FYI @darcatron , without the getter here, this value is not available when rendering the template, so `{{ shortenSlaveUsageHostname }}` would come back empty in the rendered template